Container/CSS/avatar.css

```diff
@@ -0,0 +1,135 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: content-box;
+    font-size: 1rem;
+}
+
+.avatar-round {
+    display: flex;
+    gap: 1rem;
+    align-items: center;
+}
+
+.avatar-rounded_xS {
+    height: 3rem;
+    border-radius: 50%;
+}
+
+.avatar-rounded_S {
+    height: 3.5rem;
+    border-radius: 50%;
+}
+
+.avatar-rounded_M {
+    height: 4rem;
+    border-radius: 50%;
+}
+
+.avatar-rounded_L {
+    height: 4.5rem;
+    border-radius: 50%;
+}
+
+.avatar-square {
+    display: flex;
+    gap: 1rem;
+    align-items: center;
```

Comment:
Suggestion:
.avatar-square and .avatar-round is using same flex styles, replacing it with common class name will be good.

Reply:
resolved